integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md

```diff
@@ -97,13 +97,13 @@ Afterward, you will have a model that achieves roughly 28.7 mAP on the COCO data
 
 (You can also download a pre-trained model with a DarkNet-53 backbone from the YOLACT repository; in that case skip step 1.)
 
-1. The training command for training from scratch, with a default batch_size of 8:
-   
-   ```bash
-   python train.py --config yolact_darknet53_config 
-   ```    
-    The weights are stored in the `./weights` directory by default and use `<config>_<epoch>_<iter>.pth` naming 
-    convention
+1. The training command for training from scratch, with a default batch_size of 8. Grab an imagenet-pretrained backbone and put it in `./weights`.
+   - For Darknet53, download `darknet53.pth` from [here](https://drive.google.com/file/d/17Y431j4sagFpSReuPNoFcj9h7azDTZFf/view?usp=sharing).
+      ```bash
+      python train.py --config yolact_darknet53_config 
```

Comment:
is this the comand that downloads the weights?

Reply:
No , users need to download `darknet53.pth` from the given link and then this command initializes rest of the network based off of the backbone. And kick starts training

Once the initial training is over they would get a `yolact_draknet53_54_800000.pth` checkpoint from where they can start applying recipes.

Reply:
can you make these instructions more clear then? let's assume everyone uses darknet, and say to download the darknet53.pth file and move it to the `./weights` directory